packages/server/src/api/notification/notification-alert.controller.ts

```diff
@@ -0,0 +1,314 @@
+        return isEligibleForNotification(setting);
+      });
+
+    // for each user we need to see if it is time to fire an alert and calculate the values
+    for (const setting of userSettings) {
+      const alertData: AlertData = {};
+      const minutes = getMinutesSinceLastNotification(setting);
+      const rows = await Observation.createQueryBuilder('observation')
+        .select('COUNT(observation.unit) as count, observation.unit')
+        .where('observation.report_schema_org = :org', { org: req.appOrg!.id })
+        .andWhere(`observation.report_schema_id = 'es6ddssymptomobs'`)
+        .andWhere(`observation.custom_columns->>'TalkToSomeone' = 'true'`)
+        .andWhere('observation.timestamp BETWEEN :minTime AND :maxTime', {
+          minTime: moment().subtract(minutes, 'minutes'),
+          maxTime: moment(),
+        })
+        .groupBy('observation.unit')
+        .getRawMany();
+      console.log(rows);
+      rows.forEach((row: any) => {
+        // only add counts for units where the threshold is exceeded
+        if (row.count >= setting.threshold) {
+          alertData[row.unit] = row.count;
+        }
+      });
+
+      // only return records with actual alert data
+      if (!_.isEmpty(alertData)) {
+        output.push({
+          userSetting: setting,
+          alertData,
+        });
+      }
+    }
+    res.json(output);
+  }
+
+  // return the number of users per unit requiring immediate medical attention
+  async getUsersRequiringMedicalAttentionByUnit(req: ApiRequest<OrgSettingParams>, res: Response) {
+    const output: OutputRecord[] = [];
+
+    // first get all the users from the org who are registered for the given alert
+    const userSettings = (await getUsersRegisteredForNotification(req.appOrg!.id, 'medicalAttentionAlert'))
+      .filter((setting: UserNotificationSetting) => {
+        return isEligibleForNotification(setting);
+      });
+
+    // for each user we need to see if it is time to fire an alert and calculate the values
+    for (const setting of userSettings) {
+      const alertData: AlertData = {};
+      const minutes = getMinutesSinceLastNotification(setting);
+      const rows = await Observation.createQueryBuilder('observation')
+        .select('COUNT(observation.unit) as count, observation.unit')
+        .where('observation.report_schema_org = :org', { org: req.appOrg!.id })
+        .andWhere(`observation.report_schema_id = 'es6ddssymptomobs'`)
+        .andWhere('observation.timestamp BETWEEN :minTime AND :maxTime', {
+          minTime: moment().subtract(minutes, 'minutes'),
+          maxTime: moment(),
+        })
+        .andWhere(
+          `((observation.custom_columns->>'Score')::int >= 20 
+          OR (observation.custom_columns->>'TemperatureFahrenheit')::int >= 102
+          OR observation.custom_columns->>'Category' In (:...keywords))`,
+          {
+            keywords: ['medium', 'high', 'very High'],
```

Comment:
I believe these should all be lower case and the result from `Category` should be wrapped in `LOWER` to ensure that differences in casing don't let things fall through the cracks.  There seems to be inconsistency in the sample documents, so it's best to play it safe.